Description: Remove wacom support
 Temporary patch to resolve GNOME 49 changes that causes the desktop
 to crash.
 Upstream are looking at this and in a future update hopefully
 a fix can be made.
Author: David Mohammed <fossfreedom@ubuntu.com>
Origin: other
Bug: https://github.com/BuddiesOfBudgie/budgie-desktop/pull/739
Forwarded: not-needed
Last-Update: 2025-08-22

--- budgie-desktop-10.9.2.orig/src/session/meson.build
+++ budgie-desktop-10.9.2/src/session/meson.build
@@ -23,7 +23,6 @@ gnome_session_324_components = [
     'org.gnome.SettingsDaemon.Sharing',
     'org.gnome.SettingsDaemon.Smartcard',
     'org.gnome.SettingsDaemon.Sound',
-    'org.gnome.SettingsDaemon.Wacom',
     'org.gnome.SettingsDaemon.XSettings',
 ]
 
